### DISCUSSION TITLE
@pafuhana1213 will there be a version compatible with Mac in the future?

### DISCUSSION THREAD

**Discussion by mama2222222** [pafuhana1213/KawaiiPhysics#114] [Q&A]
Your work is really impressive. I'm curious, will there be a version compatible with Mac in the future?

**Comment by pafuhana1213** [comment]
Hi @mama2222222 ,
Unfortunately,  I don't the plan because I don't have any Mac.

Therefore, if you need the Mac version, would you please build it yourself?
thanks

**Comment by mama2222222** [comment]
thanks for your reply, I wish I could build it myself, however, I know
nothing about C++🤣.

pafuhana1213 ***@***.***> 于2024年3月27日周三 22:35写道：
